<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayForm</class>
 <widget class="QWidget" name="PlayForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放界面</string>
  </property>
  <widget class="QSlider" name="play_slider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loop_checkbox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>循环播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
